sql/core/src/test/scala/org/apache/spark/sql/SparkSessionExtensionSuite.scala

```diff
@@ -95,6 +95,12 @@ class SparkSessionExtensionSuite extends SparkFunSuite {
     }
   }
 
+  test("SPARK-37518: inject a early scan push down rule") {
```

Comment:
nit: I think we only require JIRA id for bug fixes and regressions?

Reply:
Just references https://github.com/apache/spark/blob/a3bc014d8a8d88a11d4909e13605c9b44786c3ca/sql/core/src/test/scala/org/apache/spark/sql/SparkSessionExtensionSuite.scala#L92